tools/internal_ci/macos/grpc_run_bazel_c_cpp_tests.sh

```diff
@@ -38,6 +38,7 @@ echo "${BAZEL_INVOCATION_ID}" >"${KOKORO_ARTIFACTS_DIR}/bazel_invocation_ids"
 tools/bazel \
   --bazelrc=tools/remote_build/mac.bazelrc \
   test \
+  --define=use_strict_warning=true \
```

Comment:
this is not the right place to add  flags like this.
This script only has the arguments that take care of uploading the test results to the resultstore UI and that sets up kokoro integration.

This file is where you should set compilation options is here: https://github.com/grpc/grpc/blob/master/tools/remote_build/mac.bazelrc

Reply:
This options should be enabled only when running on kokoro because this goes well with certain versions of clang. But bazelrc works on mac all the times?

Reply:
That's a misunderstanding. Note that the `mac.bazelrc` file is under `remote_build` directory (which indicates it's only used for RBE-like builds on kokoro).
The mac.bazerc file isn't a "standard" rc file that gets included in a build automatically, you need to explicitly apply it with `--bazelrc=tools/remote_build/mac.bazelrc" which is exactly what's being done here: https://github.com/grpc/grpc/blob/6f339894f18e75049a2d17a5ef233ae29555760c/tools/internal_ci/macos/grpc_run_bazel_c_cpp_tests.sh#L39

Therefore my previous concern is still valid and the `--define=use_strict_warning=true` flag still needs to be moved to mac.bazelrc.

Reply:
Oh I see. Thank you for clarification! I'm updating the PR accordingly.